src/azul/__init__.py

```diff
@@ -81,6 +82,12 @@ def cache_per_thread(f, /):
 mutable_furl = furl
 
 
+class DocumentType(Enum):
+    contributions = auto()
+    aggregates = auto()
+    replicas = auto()
```

Comment:
Nit: It seems like these should be singular instead of plural. While the plural version makes some sense in the context of an index, e.g.
```
IndexName(…, doc_type=DocumentType.contributions)
```

it is also a property in `DocumentCoordinates` resulting in a singular / plural mismatch when the document type of a single contribution is being checked, e.g.
```
assert self.coordinates.doc_type is DocumentType.contributions
```